rpmlint/checks/AlternativesCheck.py

```diff
@@ -143,6 +148,20 @@ def _check_ua_presence(self, pkg):
             return True
         return False
 
+    def _check_libalternatives_presence(self, pkg):
+        """
+        Check if there is libalternatives scriptlet present
+        """
+        # first check just if we have anything in /usr/share/libalternatives/
+        for path in pkg.files:
+            if path.startswith('/usr/share/libalternatives/'):
+                return True
+        # then check if package -alts- is required
```

Comment:
Are you seeking for `*-alts-*` or `alts` as package name? The comment is a bit misleading.